### ISSUE TITLE
Add network fund info under governance tab

### ISSUE THREAD

**Issue by jzbz** [decred/decrediton#1665]
I've been asked by several people recently where they can see the balance of the project treasury. Including the balance of the network fund and a link to it on dcrdata would be a useful feature in my view.

**Comment by xaur** [comment]
Agreed, financial info like this helps stakeholders with their decisions making. Could be useful in CLI tools too.

**Comment by alexlyp** [comment]
`curl -s https://explorer.dcrdata.org/api/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx/totals -o - | jq '.dcr_unspent'`

This will return the current dev subsidy balance

**Comment by jzbz** [comment]
This has actually been addressed and is on the bottom of the page now so I should have closed the issue.

**Comment by alexlyp** [comment]
@jz afaik this has not been implemented in decrediton yet.

**Comment by jzbz** [comment]
My bad, I thought I was commenting on dcrdata.

**Comment by szpasztor** [comment]
@alexlyp I could take this issue - can you please add extra guidance as it's one of my first issues? Basically any info that's not in the decred docs already or steps to complete.

**Comment by alexlyp** [comment]
yup go for it.  

So first I'd add a call in politeiapi to GET the subsidy from `https://explorer.dcrdata.org/api/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx/totals`

Then once you have that call returning what you'd like you can add it to the getStartupWalletInfo() in ClientActions.js so it can be available to add to state (which will then let you add a selector and use in the component of your choice).

So start with the new restful call, and add it to that startup function. Open a PR with that completed (label it WIP) and then we can go from there to add to the Governance header.

**Comment by matheusd** [comment]
You can also leave it for the second round of changes, but given this is the first access we'll add to dcrdata, we'll also need a new privacy option to enable access to it.
